lib/helpers.coffee

```diff
@@ -31,7 +31,9 @@ module.exports = Helpers =
         else
           resolve(data.stderr.join(''))
       if isNodeExecutable
-        spawnedProcess = new BufferedNodeProcess({command, args, stdout, stderr, exit})
+        options.env = JSON.parse(JSON.stringify(process.env))
```

Comment:
I saw that in the bufferedNodeProcess code, but for some reason, Object.create(process.env) returns an empty object when I use it here.